src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,4 @@
+export default function ErrorPage(){
+    return <> <h1>Sorry, this page doesn't seem to exist</h1>
+    <p>Please check the URL </p> </>
+}
```

Comment:
The nav renders with this implying I can get away from this error page, but the link to all articles doesn't seem to be working

Reply:
This bug only happens if I go to a topic that doesn't exist which is bizarre